src/DataFixtures/AppFixtures.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace App\DataFixtures;
+
+use Doctrine\Bundle\FixturesBundle\Fixture;
+use Doctrine\Persistence\ObjectManager;
+
+class AppFixtures extends Fixture
+{
+    public function load(ObjectManager $manager): void
+    {
+        // $product = new Product();
```

Comment:
Should we remove these comments?

Reply:
It's autogenerated, but I agree, removed it.